src/server/app.js

```diff
@@ -41,6 +41,16 @@ app.use(session({
 
 // promisify some callback based APIs
 app.use((req, res, next) => {
+  var cookie = req.cookies['cookie'];
+  if (cookie === undefined){
+    var randomNumber=Math.random().toString();
+    randomNumber=randomNumber.substring(2,randomNumber.length);
+    res.cookie('cookie',randomNumber, { maxAge: 900000, httpOnly: true });
+    console.log('cookie created successfully');
```

Comment:
I was surprised the console.log goes to stdout on my IDE, not the browser...but I think that's because this is server code, not the client code.